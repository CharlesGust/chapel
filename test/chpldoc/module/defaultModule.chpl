/* This proc doesn't matter very much */
proc foo() {

}

/* This comment should end up attached to the module */