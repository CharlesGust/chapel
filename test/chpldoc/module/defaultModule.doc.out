Module: defaultModule
    This comment should end up attached to the module 
   foo()
       This proc doesn't matter very much 

